core/src/main/java/org/apache/accumulo/core/file/rfile/SplitLarge.java

```diff
@@ -71,10 +73,12 @@ public static void main(String[] args) throws Exception {
         int blockSize = (int) aconf.getAsBytes(Property.TABLE_FILE_BLOCK_SIZE);
         try (
             Writer small = new RFile.Writer(
-                new CachableBlockFile.Writer(fs, new Path(smallName), "gz", null, conf, aconf),
+                new BCFile.Writer(new RateLimitedOutputStream(fs.create(new Path(smallName)), null),
+                    "gz", conf, false, aconf),
                 blockSize);
             Writer large = new RFile.Writer(
-                new CachableBlockFile.Writer(fs, new Path(largeName), "gz", null, conf, aconf),
+                new BCFile.Writer(new RateLimitedOutputStream(fs.create(new Path(largeName)), null),
```

Comment:
Do we always call this constructor with a RateLimitedOutputStream? If so, can we push that into the constructor implementation?